dockstore-webservice/src/main/java/io/dockstore/webservice/core/Version.java

```diff
@@ -44,8 +44,8 @@
 
 import com.fasterxml.jackson.annotation.JsonIgnore;
 import com.fasterxml.jackson.annotation.JsonProperty;
-import com.google.common.collect.ComparisonChain;
-import com.google.common.collect.Ordering;
+// import com.google.common.collect.ComparisonChain;
+// import com.google.common.collect.Ordering;
```

Comment:
Would remove instead of commenting out, but see comment on hashCode in this file; I think you need these back.